src/gwt/src/org/rstudio/core/client/widget/ModifyKeyboardShortcutsWidget.java

```diff
@@ -1465,6 +1470,8 @@ public Object getKey(KeyboardShortcutEntry item)
    private Pair<Integer, Integer> lastSelectedIndices_;
    private boolean swallowNextKeyUpEvent_;
 
+   private static final String RESET_BUTTON_ID = "rstudio_kybrd_shrtcts_rst";
```

Comment:
For consistency, should define the constant in `ElementIds.java` and use the appropriate `assignElementId()` overload to apply it.

Someday we should decentralize this so it's not all in this single source file but until we do let's follow the same pattern.